package/MDAnalysis/lib/formats/xdrlib.pyx

```diff
@@ -321,6 +382,7 @@ cdef class TRRFile(_XDRFile):
         """read byte offsets from TRR file directly"""
         if not self.is_open:
             return np.array([])
+        cdef int ok
```

Comment:
Ok. I thought that there'd be some overhead always converting it to a Python object.
I actually based myself off the `res` variable used in the `close` method. Then that one also doesn't need to be `cdef`, right?